Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/ComponentBroadcasterDefinition.cs

```diff
@@ -6,13 +6,13 @@
 
 namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
 {
-    internal abstract class ComponentBroadcasterDefinition
+    public abstract class ComponentBroadcasterDefinition
     {
         public abstract void EnsureComponentBroadcastersCreated(GameObject gameObject, out bool changesDetected);
         public abstract bool IsTransformBroadcasterController { get; }
     }
 
-    internal class ComponentBroadcasterDefinition<TComponentBroadcaster> : ComponentBroadcasterDefinition where TComponentBroadcaster : Component
+    public class ComponentBroadcasterDefinition<TComponentBroadcaster> : ComponentBroadcasterDefinition where TComponentBroadcaster : Component
     {
```

Comment:
if these types are now public, we need to add summaries to the public fields/functions.